src/pocketmine/entity/Entity.php

```diff
@@ -954,6 +958,15 @@ public function setHealth(float $amount) : void{
 
 		if($amount <= 0){
 			if($this->isAlive()){
+				if($this instanceof Player and $this->spawned === true and $this->hasTotemInHand()){
```

Comment:
`$this->spawned` check is here to avoid getting Item in hand on null

Reply:
=== true isn't needed